static/themes/dark.css

```diff
@@ -0,0 +1,76 @@
+::selection {
+  background-color: #004daa !important;
+  color: #e8e6e3 !important;
+}
+
+a,
+header nav a {
+  color: rgb(175, 169, 158);
+  text-decoration-color: currentcolor;
+}
+
+a:hover {
+  color: rgb(255, 147, 40);
+}
+
+header {
+  background-color: rgba(24, 26, 27, 0.95) !important;
+  box-shadow: rgba(0, 0, 0, 0.05) 0px 2px 15px 2px !important;
+}
+
+header a {
+  color: rgb(175, 169, 158);
+}
+
+.svelte-select > .svelte-select-list {
+  background-color: rgb(53, 57, 59) !important;
+}
+
+/* All buttons */
+.button.a {
+  background-color: rgb(182, 44, 10) !important;
+  color: rgb(232, 230, 227) !important;
+  text-decoration-color: currentcolor !important;
+}
+
+.button.a:hover {
+  transform: scale(0.99);
+  filter: brightness(0.9);
+  background-color: rgb(164, 40, 9);
+  color: rgb(209, 207, 204);
+}
+
+main > section {
+  background-color: rgb(53, 57, 59) !important;
+}
+
+div.wrapper-content {
+  background-image: linear-gradient(rgba(24, 26, 27, 0.5), rgba(24, 26, 27, 0.5)),
+    url("/mambi-icon-oulined.png") !important;
+}
+
+.window {
+  border-color: rgb(129, 120, 106) !important;
+  background-color: rgb(24, 26, 27) !important;
+}
```

Comment:
This style is too specific for a component.
We should define the style for a component within the component file. The same of the other components here.